### ISSUE TITLE
Error after switching branches: TypeError: Cannot read property 'dependencies' of null at walkDependencyBlock

### ISSUE THREAD

**Issue by pbeshai** [mzgoddard/hard-source-webpack-plugin#14]
I added some new files in my branch, committed them then switched back to master (my branch has yet to be merged) and this error occurred:

```
Unhandled rejection TypeError: Cannot read property 'dependencies' of null
    at walkDependencyBlock (/my/project/node_modules/hard-source-webpack-plugin/index.js:233:16)
    at /my/project/node_modules/hard-source-webpack-plugin/index.js:250:11
    at Array.forEach (native)
    at /my/project/node_modules/hard-source-webpack-plugin/index.js:242:34
    at tryCatcher (/my/project/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/my/project/node_modules/bluebird/js/release/promise.js:504:31)
    at Promise._settlePromise (/my/project/node_modules/bluebird/js/release/promise.js:561:18)
    at Promise._settlePromise0 (/my/project/node_modules/bluebird/js/release/promise.js:606:10)
    at Promise._settlePromises (/my/project/node_modules/bluebird/js/release/promise.js:681:18)
    at Async._drainQueue (/my/project/node_modules/bluebird/js/release/async.js:138:16)
    at Async._drainQueues (/my/project/node_modules/bluebird/js/release/async.js:148:10)
    at Immediate.Async.drainQueues [as _onImmediate] (/my/project/node_modules/bluebird/js/release/async.js:17:14)
    at tryOnImmediate (timers.js:543:15)
    at processImmediate [as _immediateCallback] (timers.js:523:5)
```

**Comment by pbeshai** [comment]
Sorry this is on 0.0.22, so it might already be fixed...

**Comment by trotzig** [comment]
Yeah I believe this problem is fixed in 0.0.23.
